src/IndexingWrapper.cpp

```diff
@@ -158,16 +158,17 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
 	    int colNo;
 
 	    if (list.containsElementNamed("prefix")) {
-		try {
+		
 		    if (list.containsElementNamed("index")) {
 			colNo = Rcpp::as<int>(list["index"]);
-		    } else {
+		    } 
+		    else if (list.containsElementNamed("name")){
```

Comment:
You're slipping into inconsistency again: the `if` line has a space before the `{` but the `else if` line does not. You also want to reindent lines when you remove a surrounding block, as you have done here. (If you add `?w=1` onto the end of github diff URLs, it will ignore whitespace changes which can help check the commit is doing the right thing.)

I'd also be inclined to collapse onto one line `} else if (…) {`, which is a more common layout.